### DISCUSSION TITLE
Render docusaurus-flavored markdown in separate React app

### DISCUSSION THREAD

**Discussion by sklinov** [facebook/docusaurus#8300] [Q&A]
In our project, we have a free standing Docusaurus instance, but at the same time we need some of those pages to be rendered the same way in our react application.
Is there a npm module for this? Similar to `react-markdown` that I can pass my markdown string into.
Thanks!

**Comment by slorber** [comment]
It's not clear to me what the end result should look like.

Please give me an example docusaurus document, and show me how it's supposed to render after your integration. You can create the desired HTML manually, or use a mockup tool like excalidraw.

"rendered the same way" is not clear enough to me to understand.

Might be related to https://github.com/facebook/docusaurus/issues/7480
